docs/docsite/rst/network/user_guide/platform_nxos.rst

```diff
@@ -0,0 +1,125 @@
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Enable Mode**         | | not supported by NXOS                       | | not supported by NXOS                 |
+| | (Privilege Escalation)  | |                                             |                                         |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| **Returned Data Format**  | ``stdout[0].``                                | ``stdout[0].messages[0].``              |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+
+
+Using CLI in Ansible 2.5
+================================================================================
+
+Example CLI ``group_vars/nxos.yml``
+-----------------------------------
+
+.. code-block:: yaml
+
+   ansible_connection: network_cli
+   ansible_network_os: nxos
+   ansible_user: myuser
+   ansible_ssh_pass: !vault...
+   ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q bastion01"'
+
+
+- If you are using SSH keys (including an ssh-agent) you can remove the ``ansible_ssh_pass`` configuration.
+- If you are accessing your host directly (not through a bastion/jump host) you can remove the ``ansible_ssh_common_args`` configuration.
+- If you are accessing your host through a bastion/jump host, you cannot include your SSH password in the ``ProxyCommand`` directive. To prevent secrets from leaking out (for example in ``ps`` output), SSH does not support providing passwords via environment variables.
+
+Example CLI Task
+----------------
+
+.. code-block:: yaml
+
+   - name: Backup current switch config (nxos)
+     nxos_config:
+       backup: yes
+     register: backup_nxos_location
+     when: ansible_network_os == 'nxos'
+
+
+
+Using NX-API in Ansible 2.5
+================================================================================
+
+Enabling NX-API
+--------------
+
+Before you can use NX-API to connect to a switch, you must enable NX-API. To enable NX-API on a new switch via Ansible, use the ``nxos_nxapi`` module via the CLI connection. Set up group_vars/nxos.yml just like in the CLI example above, then run a playbook task like this:
+
+.. code-block:: yaml
+
+   - name: Enable NX-API
+      nxos_nxapi:
+          enable_http: yes
+          enable_https: yes
+      when: ansible_network_os == 'nxos'
+
+To find out more about the options for enabling HTTP/HTTPS and local http see the :ref:`nxos_nxapi <nxos_nxapi>` module documentation.
+
+Once NX-API is enabled, change your ``group_vars/nxos.yml`` to use the NX-API connection.
+
+Example NX-API ``group_vars/nxos.yml``
+-------------------------------------
+
+.. code-block:: yaml
+
+   ansible_connection: local
+   ansible_network_os: nxos
+   ansible_user: myuser
+   ansible_pass: !vault... 
```

Comment:
`ansible_ssh_pass`

Reply:
Thanks for catching that. Fixed in 793140ded6f7f5357d413d17ec82db9b7d4bed7f.